### ISSUE TITLE
The doc of `linalg.vector_norm()` should say there is `x` or `input` parameter

### ISSUE THREAD

**Issue by hyperkai** [pytorch/pytorch#136560]
### 📚 The doc issue

[The doc](https://pytorch.org/docs/stable/generated/torch.linalg.vector_norm.html) of `linalg.vector_norm()` says there is `x` parameter as shown below:

> torch.linalg.vector_norm(x, ord=2, dim=None, keepdim=False, *, dtype=None, out=None) → [Tensor](https://pytorch.org/docs/stable/tensors.html#torch.Tensor)

> x ([Tensor](https://pytorch.org/docs/stable/tensors.html#torch.Tensor)) – tensor, flattened by default, but this behavior can be controlled using dim.

Then, `x` parameter works and `input` parameter also works as shown below:

```python
import torch
from torch import linalg

my_tensor = torch.tensor([-2., -1., 0., 1., 2., 3.])

linalg.vector_norm(x=my_tensor) # tensor(4.3589)
linalg.vector_norm(input=my_tensor) # tensor(4.3589)
```

### Suggest a potential alternative/fix

[The doc](https://pytorch.org/docs/stable/generated/torch.linalg.vector_norm.html) of `linalg.vector_norm()` should say there is `x` or `input` parameter as shown below:

> torch.linalg.vector_norm(x or input, ord=2, dim=None, keepdim=False, *, dtype=None, out=None) → [Tensor](https://pytorch.org/docs/stable/tensors.html#torch.Tensor)

> x or input ([Tensor](https://pytorch.org/docs/stable/tensors.html#torch.Tensor)) – tensor, flattened by default, but this behavior can be controlled using dim.

**Comment by ezyang** [comment]
Ditto here, send us a PR!

**Comment by thomasXwang** [comment]
Hi, I’ve submitted a PR to clarify the alias usage for the x parameter in the norm function documentation
